### PR TITLE
Paying for College: Remove disclosure feedback assets

### PR DESCRIPTION
**Pull request by anselmbradford**
Paying for College feedback form used a 2000+ line CSS file. This deletes that asset and uses existing CSS on the page.

## Changes

- Remove static directory in paying_for_college app and move assets to unprocessed/apps/paying-for-college.

## How to test this PR

1. Check the following pages against production:
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/test/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/feedback/
2. JS on http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/test/  should still work.

## Screenshots

Before:

Screen Shot 2021-03-05 at 6 16 58 PM

After:

Screen Shot 2021-03-05 at 6 17 12 PM

Before:
Screen Shot 2021-03-05 at 6 06 43 PM

After:
Screen Shot 2021-03-05 at 6 07 06 PM

## Notes and todos

- I know after looks a little worse, but once these templates migrate off the `base_update` template and use our main site's `base` template, the layout will improve. For instance, they'll pick up paragraph's max-width value.

### CODE REVIEW

File: cfgov/paying_for_college/forms.py [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 
 class FeedbackForm(forms.Form):
-    message = forms.CharField(widget=forms.Textarea)
+    message = forms.CharField(widget=forms.Textarea(attrs={
+            'class': 'a-text-input',
+        }))
```
Comment (Scotchester) [inline cfgov/paying_for_college/forms.py:7]: Deintenting both of these lines by 4 spaces each should fix the linting issue.

File: cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html [hunk 2]
```diff
@@ -25,19 +19,23 @@ <h1>Paying for College</h1>
...
+              <h4>What do you think about this tool?</h4>
           </header>
           <div>
               <p>The Consumer Financial Protection Bureau is a 21st century agency committed to using technology to empower consumers to take control over their economic lives.</p>
               <p>With your help, you can help us design a tool that makes that process a little bit less daunting &ndash; helping families make smarter choices about student debt.</p>
-              <form method="post" id="feedback-container">
-                  <label for="id_message"><span class="demi">Enter your comments here</span></label>
+              <form method="post" id="feedback-container" class="block">
+                  <p><label for="id_message"><strong>Enter your comments here</strong></label></p>
                                   {% csrf_token %}
```
Comment (Scotchester) [inline cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html:29]: Let's fix this indentation while we're here.

```suggestion
                  {% csrf_token %}
```

File: cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html [hunk 3]
```diff
@@ -25,19 +19,23 @@ <h1>Paying for College</h1>
       <div class="col col7 suf5">
           <hr>
           <header>
-              <h2>What do you think about this tool?</h2>
+              <h4>What do you think about this tool?</h4>
```
Comment (Scotchester) [inline cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html:22]: What's the reason for this change? It will introduce a skipped-heading-levels accessibility error.

File: cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html [hunk 4]
```diff
@@ -25,19 +19,23 @@ <h1>Paying for College</h1>
...
               <p>With your help, you can help us design a tool that makes that process a little bit less daunting &ndash; helping families make smarter choices about student debt.</p>
-              <form method="post" id="feedback-container">
-                  <label for="id_message"><span class="demi">Enter your comments here</span></label>
+              <form method="post" id="feedback-container" class="block">
+                  <p><label for="id_message"><strong>Enter your comments here</strong></label></p>
                                   {% csrf_token %}
-                  <span class="pfc-feedback">{{ form.message }}</span>
-                  <p><button>Submit your feedback</button></p>
+                  <p class="pfc-feedback">{{ form.message }}</p>
+                  <p><button class="a-btn">Submit your feedback</button></p>
```
Comment (Scotchester) [inline cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html:31]: These really shouldn't be in paragraph tags, but I guess it's fine until we migrate off of this template and do further cleanup.

**Review by Scotchester — CHANGES_REQUESTED** [review]
A few bits and bobs that need fixing.

File: cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html (hunk 3 above)
Comment (anselmbradford) [reply cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html:22]: I was just matching the prior design heading size, but I can keep it an h2 and it'll be larger.

**Review by higs4281 — APPROVED** [review]
Works as expected, and I confirmed that feedback gets stored to the database with fields intact. Thanks for this cleanup!

**Review by Scotchester — APPROVED** [review]
